view/frontend/templates/express/checkout-button.phtml

```diff
@@ -18,12 +16,12 @@ $formSelector = $viewModel->getFormSelector($this->getFormSelector(), $block->ge
     <?= $viewModel->getStyleCss() ?>
 </style>
 <?php if ($viewModel->shouldRender($triggerContext)): ?>
-    <div class="amwal-express-checkout-button<?= $triggerContext === ExpressCheckoutButton::TRIGGER_CONTEXT_MINICART ? ' hidden' : '' ?> <?= $escaper->escapeHtmlAttr($triggerContext) ?>"
-         data-trigger-context="<?= $escaper->escapeHtmlAttr($triggerContext) ?>"
-         data-locale="<?= $escaper->escapeHtmlAttr($viewModel->getLocale()) ?>"
-         data-scope-code="<?= $escaper->escapeHtmlAttr($viewModel->getStoreCode()) ?>"
-         <?= $formSelector ? 'data-form-selector="' . $escaper->escapeHtmlAttr($formSelector) . '"' : '' ?>
-         id="<?= $escaper->escapeHtmlAttr($buttonId) ?>"
+    <div class="amwal-express-checkout-button<?= $triggerContext === ExpressCheckoutButton::TRIGGER_CONTEXT_MINICART ? ' hidden' : '' ?> <?= $triggerContext ?>"
```

Comment:
Do we really need to remove the escaper? If it's not available because of a Magento version constraint we should instead move back to using the `$block` for escaping (eg. `$block->escapeHtmlAttr(....)`)